### ISSUE TITLE
Allow organizational admins to manage subscriptions

### ISSUE THREAD

**Issue by TylerFisher** [news-catalyst/presspass-frontend#13]
If you are an admin of an organization, you should be able to manage what plans that organization subscribes to.

**Comment by jacqui** [comment]
Hey @TylerFisher does this permission restriction mean I need to check on the actual `ManageOrganizationPage` itself that the currently logged in user is an admin of the org?

This is currently checked on the `ProfilePage` in the `MembershipCard` - the 'manage' button only appears if you're an admin. I am not 100% on what checks happen in the API.

I guess to pull this off on the 'manage org' page, I'd have to:

* `ensureMembershipsForUser(props.actions, uuid, props.memberships);` (in addition to the `ensureOrganizations` call that's happening)
* find the matching org in the memberships
* check if the admin flag is true
* if it is, render the page as usual
* if it's false, redirect to profile

What do you think? The work done so far is in PR #16

**Comment by TylerFisher** [comment]
Yeah, I think we have to do everything possible to ensure people who aren't supposed to reach this page can't reach it.

**Comment by TylerFisher** [comment]
Okay, I think in the commotion of the week I explained this poorly and forgot how this is actually supposed to work.

Instead of getting the `plan` off of the organization (which is just how MuckRock works right now), you'll need to get subscriptions from the API (`/pp-api/organizations/{organization_uuid}/subscriptions/`). You can POST to that API to add subscriptions. People can subscribe to multiple plans.

I know you asked me about this and I gave you the wrong answer because the existing `plan` confused me. Sorry.

**Comment by jacqui** [comment]
I'm redoing this work today using `subscriptions` instead of the single `plan` - no worries :)

**Comment by jacqui** [comment]
Hey @TylerFisher, I've made some progress today on managing plans... but I have some questions. Can we chat about this tomorrow?

Here's what I've done:

* removed the string `plan` from the `Organization` type
* added scaffolding around `Subscription` (api, actions, etc)
* request a list of all subscriptions for an org on the Manage Org page
  * render a list of all plans on the "Manage Organization" page with checkboxes indicating which the org already subscribes to
* `createSubscription`

The last bit is where I had some trouble: this create-subscription API endpoint doesn't have much documentation, so I was trying to figure it out via the code. I think it accepts a `{plan: PK}` payload, where `PK` is a number, primary key for the plan.

I got it to work on one plan: `{plan: 1}`:

* some choices result in this error `["Invalid pk "2" - object does not exist."]` - I'm not sure why?
* other plans error with `["Must supply a credit card for paid plan"]`

Also, I had to add `id` values to my local API responses for this to work. Our front-end expects IDs but the API doesn't always return them. I'm not clear on whether this is okay to add to the API though :-/ 

To summarize:

* I think we should chat about how this feature should work, even for an initial version
* I think an overview of the squarelet API would be _very_ helpful for me

Thank you :)
